aries-core/src/js/components/helpers/Anchors/Anchor.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Anchor as GrommetAnchor, Box, Text } from 'grommet';
+
+export const Anchor = ({ label, size, weight, ...rest }) => {
```

Comment:
For documentation purposes, as [discussed previously](https://github.com/hpe-design/aries/pull/106#discussion_r357422499), we plan to refactor Aries' anchor component with a `type` property once design executions for anchors are fleshed out further.